### ISSUE TITLE
config backup not saving tasks

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#770]
I don't know if this is by design but just adding in case.

To reproduce I created a config backup and downloaded the file, upon unzipping there was no info re the existing tasks on the system whose config was backed up.

**Comment by phillxnet** [comment]
OK as per documentation updates at http://rockstor.com/docs/config_backup.html this is currently by design so closing.

**Comment by FroggyFlox** [comment]
Hi @phillxnet,

I was going to open a new issue but Github pointed me to this closed issue. I currently have a working WIP branch to fixes this issue. Although it was closed for good reasons at the time, should we re-open this one or would you prefer me to open a new issue for that?
What the WIP branch resolves is pretty much identical to this issue and implements what was marked as planned in the docs: **support backup and restore of scheduled tasks**.

Let me know how you'd prefer to proceed.
Thanks!

**Comment by phillxnet** [comment]
@FroggyFlox Apologise for slow response on this one. I think the best way to move forward on this one is to open a fresh issue in your name given, as you say, this issue had it's context of a 'bug' and was closed in that context. Where as a fresh issue in your name could be more clearly presented as an additional feature. And more clearly represent attribution.

Let me know if you disagree and I'll be happy to re-open but we should take care not to second guess our selves too much as then we loose / muddle the context of issues. I.e. I opened this originally assuming it was a bug and closed it as it was documented as a planned feature: which your proposed fresh issue would then addresses more clearly.

Might give time context to link to this (erroneous) issue in your new one though as then archival search will in turn, turn up the fresh (feature add in this case) related issues.

Thanks for your diligence here.
